src/client/rsg-components/StyleGuide/StyleGuideRenderer.js

```diff
@@ -76,10 +80,19 @@ export function StyleGuideRenderer({
 			</main>
 			{hasSidebar && (
 				<div className={classes.sidebar} data-testid="sidebar">
-					<header className={classes.logo}>
+					<header className={classes.sidebarSection}>
 						<Logo>{title}</Logo>
 						{version && <Version>{version}</Version>}
 					</header>
+					{themes && (
+						<header className={classes.sidebarSection}>
```

Comment:
We shouldn't have two headers here.

```suggestion
						<section className={classes.sidebarSection}>
```